<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Afficherjoueur">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AfficherJoueur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AjouterNote">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AjouterNote</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>431</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editNote">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>280</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
